lib/global_uid/active_record_extension.rb

```diff
@@ -60,7 +60,7 @@ def global_uid_table
       end
 
       def ensure_global_uid_table
-        return @global_uid_table_exists if @global_uid_table_exists
+        return @global_uid_table_exists if defined?(@global_uid_table_exists)
```

Comment:
You're changing the logic here. The new code will allow returning a falsey value, the old code wouldn't.